charts/consul/templates/mesh-gateway-deployment.yaml

```diff
@@ -337,7 +343,7 @@ spec:
           lifecycle:
             preStop:
               exec:
-                command: ["/bin/sh", "-ec", "/consul-bin/consul services deregister -id=\"{{ .Values.meshGateway.consulServiceName }}\""]
+                command: ["/bin/sh", "-ec", "/consul-bin/consul logout"]
```

Comment:
🚗 we still need to deregister the service

Reply:
hmm. i assumed a logout implicitly performed a de-register. thanks for the shoutout!